README.md

```diff
@@ -1,2 +1,66 @@
-# go-xline
-go-xline is an official xline client sdk, written in Go.
+# Go-Xline
+
+[![Apache 2.0 licensed][apache-badge]][apache-url]
+[![codecov][cov-badge]][cov-url]
+[![Build Status][actions-badge]][actions-url]
+
+[apache-badge]: https://img.shields.io/badge/license-Apache--2.0-brightgreen
+[apache-url]: https://github.com/datenlord/Xline/blob/master/LICENSE
+[cov-badge]: https://codecov.io/gh/xline-kv/xline/branch/master/graph/badge.svg
+[cov-url]: https://codecov.io/gh/xline-kv/go-xline
+[actions-badge]: https://github.com/datenlord/xline/actions/workflows/ci.yml/badge.svg?branch=master
+[actions-url]: https://github.com/xline-kv/go-xline/actions
+
+Go-Xline is an official xline client sdk, written in Go.
+
+## Features
```

Comment:
It's better to list the client interfaces here.

Reply:
Yout're right. I've made the changes you suggested.